doc/nrf/includes/hid_provider_consumer_system_control.txt

```diff
@@ -0,0 +1,114 @@
+.. note::
```

Comment:
Can we keep the file together with nRF Desktop documentation? (as it's application specific)